packages/server/jest.config.js

```diff
@@ -9,8 +9,14 @@ module.exports = {
     }
   },
   transform: {
-    ...tsjPreset.transform
+    '\\.(gql|graphql)$': 'jest-transform-graphql',
+    ...tsJestPresets.jsWithBabel.transform
   },
   modulePaths: ['<rootDir>/packages/'],
+  moduleNameMapper: {
+    'server/(.*)': ['<rootDir>/$1'],
+    'parabol-client/(.*)': ['<rootDir>/../client/$1'],
+    '~/(.*)': ['<rootDir>/../client/$1']
+  },
```

Comment:
ts-jest does provide [a helper method](https://kulshekhar.github.io/ts-jest/docs/getting-started/paths-mapping/), but because `client` is one directory down it doesn't quite work for this use case. It was easy enough to just write this out.